src/styles/less/ParentNavTopBar.less

```diff
@@ -6,6 +6,9 @@
     margin-bottom: 2%;
     border-bottom: 1px solid gray;
     background-color: @turquoise-flat;
+    a {
+        color: black;
+    }
```

Comment:
added link styling to get past default blue text

Reply:
Looks great!